attributes/default.rb

```diff
@@ -13,3 +13,8 @@
 
 default[:mysql_component][:sql_url] = nil
 default[:mysql_component][:hosts] = ["localhost"]
+
+
+local_ipv4 = node[:cloud][:local_ipv4].kind_of?(Array) ? node[:cloud][:local_ipv4].first : node[:cloud][:local_ipv4]
```

Comment:
`node[:cloud]` does not exist if cookbook is run in Vagrant for example. If you are using this attribute - you first should check for existence.